sql/2_0_0-to-2_0_1_upgrade.sql

```diff
@@ -118,6 +118,46 @@ DROP TABLE IF EXISTS `menu_trees`;
 
 DROP TABLE IF EXISTS `menu_entries`;
 
+#IfNotTable updater_users
+CREATE TABLE `updater_users` (
```

Comment:
Why isn't this just a new entry in the Administration GACL, seems odd to have a separate table to provide a user permission when we have GACL for that.